tests/cli/explain/CallInputSpacing/sprocket_command

```diff
@@ -0,0 +1 @@
+explain CallInputSpacing
```

Comment:
The case insensitive seems like something we should test, but probably not a duplicate with the non-case insensitive test.